.github/workflows/test-php.yml

```diff
@@ -25,6 +25,8 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: "${{ matrix.php }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This may need a permission reduction though, tokens don't come with a sensible minimal set of permissions. Something like:

```
permissions:
      contents: read
```